mobile/src/components/chat-conversation/components/messages-list/messages-list.tsx

```diff
@@ -14,28 +14,39 @@ type Props = {
 };
 
 const MessagesList: FC<Props> = ({ currentUserId, messages }) => {
+  const groupedByDateMessages = groupByKey(messages, 'createdAt');
```

Comment:
Not sure if this will work, what if messages have same date but different time?

Reply:
I've also thought about that. Will make changes soon